### PR TITLE
Rename Bitwala to Nuri

### LINKED ISSUE #6059: Update Bitwala
### Site name

Nuri

### Site URL

https://nuri.com/

### Update reason

Its name has changed.

### Additional information

https://play.google.com/store/apps/details?id=com.bitwala.app

### Issue Eligibility

- [X] The issue I'm creating is not a duplicate of an existing [issue](https://github.com/2factorauth/twofactorauth/issues).
- [X] The issue I'm creating is not a duplicate of an existing [pull request](https://github.com/2factorauth/twofactorauth/pulls)

### PR DESCRIPTION
**Pull request by phallobst**
Closes #6059

### CODE REVIEW

**Review by Sami32 — COMMENTED** [review]
Oups...i should have added that they support SMS 2FA.

File: entries/n/nuri.com.json [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-  "Bitwala": {
-    "domain": "bitwala.com",
-    "url": "https://www.bitwala.com/",
+  "Nuri": {
+    "domain": "nuri.com",
+    "additional-domains": [
+      "bitwala.com"
+    ],
     "tfa": [
-      "sms",
```
Comment (ApCoder123) [inline entries/n/nuri.com.json:6]: As @Sami32 mentions, it does appear that they also support SMS 2FA - https://support.nuri.com/hc/en-gb/articles/360009149240-TAN

File: img/n/nuri.com.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg class="logo-stacked" viewBox="0 0 300 300"><rect width="300" height="300" rx="67.38" fill="#2c232e"/><path d="M129.65 189.41c0-11.58-10.09-20.17-21.67-20.17H67.65v68H77v-27.67h22.17l19.27 27.64h11.21l-19.93-27.71c10.83-.8 19.93-9.1 19.93-20.09zM108 201H77v-23.17h31c6.35 0 12.33 4.48 12.33 11.58S114.33 201 108 201zM170.35 177.83h26.15v50.79h-26.15v8.59h62v-8.59h-26.14v-50.79h26.14v-8.59h-62v8.59zM120.31 122.17h-2.62L85.2 62.79H67.65v67.97h9.34V71.38h2.61l32.49 59.38h17.56V62.79h-9.34v59.38zM223 62.79v39.59c0 13.07-8.59 22.78-21.67 22.78s-21.66-9.71-21.66-22.78V62.79h-9.34v39.59c0 17.92 13.08 31.37 31 31.37s31-13.45 31-31.37V62.79z" fill="#d3d2d2"/></svg>
```
Comment (ApCoder123) [inline img/n/nuri.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300"><rect width="300" height="300" rx="67.38" fill="#2c232e"/><path d="M129.65 189.41c0-11.58-10.09-20.17-21.67-20.17H67.65v68H77v-27.67h22.17l19.27 27.64h11.21l-19.93-27.71c10.83-.8 19.93-9.1 19.93-20.09zM108 201H77v-23.17h31c6.35 0 12.33 4.48 12.33 11.58S114.33 201 108 201zM170.35 177.83h26.15v50.79h-26.15v8.59h62v-8.59h-26.14v-50.79h26.14v-8.59h-62v8.59zM120.31 122.17h-2.62L85.2 62.79H67.65v67.97h9.34V71.38h2.61l32.49 59.38h17.56V62.79h-9.34v59.38zM223 62.79v39.59c0 13.07-8.59 22.78-21.67 22.78s-21.66-9.71-21.66-22.78V62.79h-9.34v39.59c0 17.92 13.08 31.37 31 31.37s31-13.45 31-31.37V62.79z" fill="#d3d2d2"/></svg>
```

**Review by Sami32 — CHANGES_REQUESTED** [review]
Add SMS as a tfa supported method:
https://support.nuri.com/hc/en-gb/articles/360009149240-TAN

**Review by Carlgo11 — APPROVED** [review]

**Review by Sami32 — APPROVED** [review]

**Review by ApCoder123 — APPROVED** [review]
I added another region based on these pages:
https://support.nuri.com/hc/en-gb/articles/360021661419-Which-countries-can-the-debit-card-be-shipped-to-
https://nuri.com/supported-nationalities/

**Pull request merged**